src/lib/configuration.rs

```diff
@@ -0,0 +1,43 @@
+use std::env;
+
+#[derive(Debug)]
+pub struct Configuration {
+    pub nun_user: String,
+    pub nun_pwd: String,
+    pub nun_dbs_dir: String,
+    pub nun_ws_addr: String,
+    pub nun_http_addr: String,
+    pub nun_tcp_addr: String,
+    pub nun_replicate_addr: String,
+    pub nun_log_level: String,
+    pub nun_log_dir: String,
+    pub nun_log_to_file: String,
+}
+
+#[cfg(debug_assertions)]
+fn expect_env_var(name: &str, default: &str) -> String {
+    return env::var(name).unwrap_or(String::from(default));
+}
+
+#[cfg(not(debug_assertions))]
+fn expect_env_var(name: &str, _default: &str) -> String {
+    return env::var(name).expect(&format!(
+        "Environment variable {name} is not defined",
+        name = name
+    ));
+}
+
+pub fn get_configuration() -> Configuration {
+    Configuration {
+        nun_user: expect_env_var("NUN_USER", "mateus"),
+        nun_pwd: expect_env_var("NUN_PWD", "mateus"),
```

Comment:
Can we not add default here??  I prefer to let it break if the user does not provide a user than letting a default user `mateus` hardcoded

Reply:
the default value will only be used when user is using "cargo run" / debug mode...
if not in debug mode, it will be empty. 
i will add the check if got user provided and break at main.rs instead of here in config.